sencha-workspace/packages/slate-cbl/src/view/demonstrations/SkillList.js

```diff
@@ -225,11 +225,26 @@ Ext.define('Slate.cbl.view.demonstrations.SkillList', {
                 return Slate.API.buildUrl("/cbl/dashboards/tasks/student#"+d.Student.Username+"/"+d.StudentTask.Task.Section.Code+"/"+d.StudentTask.ID);
             },
             linksExist: function(d) {
-                var submissionAttachments = d.StudentTask.Attachments.filter(a => a.status !== 'removed');
```

Comment:
this variable wasn't being used, my IDE highlighted this -- make sure your IDE is set up to highlight such things too